Documentation/workflow/UsingYourBuild.md

```diff
@@ -46,148 +54,158 @@ this by simply listing the name of the Microsoft.NETCore.Runtime.CoreCLR you bui
+that libraries between your runtime and that package are far enough apart to cause issues, so
+it is best to have the latest version of Microsoft.NETCore.App package if you are working on the
+latest version of the source in coreclr master branch. You can find the latest package by looking
+at https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.App.
+
+### Step 4: Add a RuntimeIdentifier
+In order to run with your local changes you will change your application to be a self-contained application
+and not trying to run on the shared framework. In order to do that you will need to add a `RuntimeIdentifier`
+to your project.
 
-Replace the HelloWorld\project.json with [project.json](../../tests/src/Common/netcoreapp/project.json), and update 
-`1.2.0-beta-XXXXX-X` version number in the dependencies section with the version number for your build of the runtime.
-This is the line that tells the tools that you want YOUR version of the CoreCLR runtime.
 ```
-       "Microsoft.NETCore.Runtime.CoreCLR": "1.2.0-beta-24528-0"
+  <PropertyGroup>
+    ...
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+  </PropertyGroup>
 ```
 
-The differences between the project.json generated by the tool and the replacement:
+For windows you will want `win7-x64` but for other OS's you will need to set it to the most appropriate one based
+on what you built. You can generally figure that out by looking at the packages you found in your output. In our
+example you will see there is a package with the name `runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR.2.0.0-beta-25023-0.nupkg`
+so you will want to put whatever id is between `runtime.` and `Microsoft.NETCore.Runtime.CoreCLR`.
 
-- Removed Microsoft.NETCore.App platform dependency (`"type": "platform"`). This tells the build system that you don't want to 
-use runtime and libraries that came with the dotnet.exe tool but to fetch the dependencies from the Nuget cache.  If you don't do this
-the tools will ignore your request to make the app use an explicitly specified runtime.   
-- Added the 'runtimes' line at the top level.  The runtime name includes the OS name and the architecture name
-you can find the appropriate name for your OS [here](https://github.com/dotnet/core-docs/blob/master/docs/core/rid-catalog.md).
-This tells the tools exactly which flavor of OS and processor architecture you are running on, so it can find the right
-Nuget package for the runtime.
-- Changed netcoreapp1.0 to netcoreapp1.1. This tells the tools that you want to use the latest .NET Core Framework.
-- Expanded Microsoft.NETCore.App metapackage into explicit list of the .NET Core Framework packages because of there is no good published
-build of Microsoft.NETCore.App metapackage for netcoreapp1.1 yet.
 
-### Step 4: Place your build directory and beta .NET Core Framework feed on your Nuget Path
+### Step 5: Place your build directory and beta .NET Core Framework feed on your Nuget Path
```

Comment:
I think that these complex steps with nuget packages should be marked as optional. The recommended default should be just copy over what you have built - the `Quick updates in place` step. It is what I going myself - very simple and hard to make a mistake.